docker-for-mac/release-notes.md

```diff
@@ -13,12 +13,38 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
-## Docker Desktop 3.4.0
-2021-06-09
+## Docker Desktop 3.5.0
+2021-06-23
 
 [Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn}
 [Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn}
 
+
+### New
+
+- Dev Environments
+
+### Upgrades
+- [Compose V2 beta](https://github.com/docker/compose-cli/releases/tag/v2.0.0-beta.4)
+- [Kubernetes 1.21.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.21.2)
+
+
+### Bug fixes and minor changes
+- Fix a bug where Docker fails to start because `/usr/bin` is not on the `PATH`. Fixes [docker/for-mac#5770](https://github.com/docker/for-mac/issues/5770)
+- Allow files to be modified inside a host directory which is a nested mountpoint in a container. Fixes [docker/for-mac#5748](https://github.com/docker/for-mac/issues/5748).
+- Fixed a bug where a container cannot be started when a file is bind-mounted into a nested mountpoint. Fixes [docker/compose-cli#1795](https://github.com/docker/compose-cli/issues/1795).
+- Fixed a settings migration bug which caused Docker Desktop not to find images and containers data after upgrading to 3.4.0. Fixes [docker/for-mac#5754](https://github.com/docker/for-mac/issues/5754).
+- Fix link to policy regarding uploaded diagnostics data. Fixes [docker/for-mac#5741](https://github.com/docker/for-mac/issues/5741)
+- Highlight the architecture of the non-native images in the dashboard
+- The default Unix domain socket name is `~/.docker/run/docker.sock`. We recommend using the `docker context` called `desktop-linux`. For backwards compatibility `/var/run/docker.sock` is still available.
+- Fix using virtualization.framework on macOS Monterey
+
+## Docker Desktop 3.4.0
+2021-06-09
+
+[Download for Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/65384/Docker.dmg)
+[Download for Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/65384/Docker.dmg)
```

Comment:
Should these have a UTM?

Reply:
Not required for older releases. There was a discussion with Dieu on this in https://github.com/docker/docker.github.io/pull/12832#discussion_r630192598.

Reply:
Right, I've added on windows as well.